Clients/src/presentation/pages/Home/index.tsx

```
@@ -111,18 +102,21 @@ const Home = ({ projects = mockProjects }: HomeProps) => {
+            </Suspense>
           </Stack>
-          <MetricSection
-            title="All projects compliance status"
-            metrics={complianceMetrics}
-          />
-          <MetricSection
-            title="All projects risk status"
-            metrics={riskMetrics}
-          />
+          {(["compliance", "risk"] as const).map((metricType) => (
+            <Suspense fallback={<div>Loading...</div>}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)